Form-Controls/styles.css

```diff
@@ -0,0 +1,98 @@
+* {
```

Comment:
Did you try to have focused elements? How we can do that?

Check CSS in [README](https://github.com/CodeYourFuture/Module-User-Focused-Data/blob/main/Form-Controls/README.md)